torch/testing/_internal/common_methods_invocations.py

```diff
@@ -103,17 +104,47 @@ def __init__(self, input, *, args=tuple(), kwargs=None, output_process_fn_grad=N
         # for such inputs (as they will error out otherwise).
         self.broadcasts_input = broadcasts_input
 
-    def __repr__(self):
+    def _repr_helper(self, formatter):
```

Comment:
Add comments for each of these functions explaining what they do and their intended use